test/integration/attach-volume.sh

```diff
@@ -26,7 +26,8 @@ fi
 
 export nodeName=$1
 export volumeName=$2
-export volumeId=$(kubectl get azvolume ${volumeName} -n azure-disk-csi -o yaml | egrep -i '/subscriptions/[a-z0-9_\-]+/resourceGroups/[a-z0-9_\-]+/providers/Microsoft.Compute/disks/[a-z0-9_\-]+' | awk -F' ' '{ print $2 }')
+export volumeList="kubectl get azvolume ${volumeName} -n azure-disk-csi -o yaml"
+export volumeId=$( $volumeList | egrep -i '/subscriptions/[a-z0-9_\-]+/resourceGroups/[a-z0-9_\-]+/providers/Microsoft.Compute/disks/[a-z0-9_\-]+' | awk -F' ' '{ print $2 }')
```

Comment:
This doesn't resolve the ShellCheck error becuase line 30 still executes a command and assigns the value to the variable, but the "return" value of the command is from `export` rather than `kubectl`.

Instead of doing this, I believe you can just use the command `set -euo pipefail` at the beginning of the script. This will cause immediate termination of the script if `kubectl` fails.

Reply:
Ah! I wanted to modify it to `export volumeList="$(kubectl get azvolume ${volumeName} -n azure-disk-csi -o yaml)"`. But, that still wouldnt fix the issue. I will execute the `set -euo pipefail` at the beginning of the script.

Reply:
The export over here was unwanted. 
The issue is, an export operation also generates a return value and that would mask the return values from other operations.

Removed the export.